### ISSUE TITLE
Jitsi torture with jwt token

### ISSUE THREAD

**Issue by pratik9722** [jitsi/jitsi-meet-torture#331]
If jitsi has enabled jwt token authentication then Jitsi torture should work token based authentication as well. 

We can provide the 24hr token from the bash script command

So the 1st participants go to the meeting with the token and then other participants will enter into the room.

**Comment by damencho** [comment]
Yep, this is planned to be implemented in the following weeks. I will leave it open and will close the issue once this is done.

**Comment by seb2020** [comment]
We have the same question. I have see AuthSetupConference.java but for the moment it doesn't support JWT token. Correct ?

**Comment by pratik9722** [comment]
@damencho Is token-based authentication still remaining?

**Comment by damencho** [comment]
Tests using jwt token had been added: https://github.com/jitsi/jitsi-meet-torture/blob/master/src/test/java/org/jitsi/meet/test/ModeratedRoomsTest.java
There is no global setting to allow some participants with token using command line, PRs are welcome.

**Comment by heyIshwar** [comment]
@damencho how do i test using command line?
